ui/job-view/pushes/Push.jsx

```diff
@@ -209,10 +209,10 @@ class Push extends React.PureComponent {
       // remove old versions of jobs we just fetched.
       const existingJobs = jobList.filter(job => !newIds.includes(job.id));
       // Join both lists and add test_paths property
-      const newJobList = [...existingJobs, ...jobs].map(job => ({
-        ...job,
-        test_paths: jobTypeNameToManifests[job.job_type_name] || [],
-      }));
```

Comment:
Where is the root source of the problem? What is the code that processes this list?
These two aproaches yield the same list.

Reply:
Agreed.  I created Bug https://bugzilla.mozilla.org/show_bug.cgi?id=1612022 to investigate this.